src/backend/joanie/urls.py

```diff
@@ -47,12 +47,11 @@
 router.register(
     "organizations", api_client.OrganizationViewSet, basename="organizations"
 )
-router.register("products", api_client.ProductViewSet, basename="products")
-router.register(
-    "course-product-relations",
-    api_client.CourseProductRelationViewSet,
-    basename="course-product-relations",
-)
+# router.register(
+#     "course-product-relations",
+#     api_client.CourseProductRelationViewSet,
+#     basename="course-product-relations",
+# )
```

Comment:
This comment is left there on purpose ?

Reply:
Oh no it's just a debug residue embedded in a fixup.